gke-e2e-cluster-create/0.1/samples/create-gke-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kubeconfig-pvc
+spec:
+  resources:
+    requests:
+      storage: 5Gi
```

Comment:
not sure it matters, but the kubeconfig probably won't be 5 gigs :)

Reply:
but _how do you know_